docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -43,90 +46,79 @@ $ cd tpcc
 The binaries are compiled with JAVA 13 and it is recommended to run these binaries with that version.
 {{< /note >}}
 
-## Step 2. Start your database
+### Step 2. Start your database
 
-Start the database using the steps mentioned [here](https://docs.yugabyte.com/latest/quick-start/create-local-cluster/macos/).
+Start your YugabyteDB cluster by following the steps [here](../../deploy/manual-deployment/).
 
-## Step 3. Configure connection properties
+{{< tip title="Tip" >}}
+You will need the IP addresses of the nodes in the cluster for the next step.
+{{< /tip>}}
 
-Set the following connection configurations in the workload configuration file (`config/workload_all.xml`).
+### Step 3. Running the TPC-C benchmark
 
-```sh
-<!-- Connection details -->
-<dbtype>postgres</dbtype>
-<driver>org.postgresql.Driver</driver>
-<DBUrl>jdbc:postgresql://<ip>:5433/yugabyte</DBUrl>
-<username>yugabyte</username>
-<password></password>
-<isolation>TRANSACTION_REPEATABLE_READ</isolation>
-```
-
-The details of the workloads have already been populated in the sample configuration files located in the `/config` directory.
-The workload descriptor works the same way as it does in the upstream branch and details can be found in the [online documentation](https://github.com/oltpbenchmark/oltpbench/wiki).
+Use the provided utility script (`./tpccbenchmark`) to run the TPC-C benchmark.
 
-## Step 4. Running the TPC-C benchmark
-
-Use the provided utility script (`./tpccbenchmark`) to run the TPC-C benchmark. Available options are:
+Before starting the workload, you will need to load the data first.
 
-```
--c,--config &lt;arg&gt;            [required] Workload configuration file
-   --clear &lt;arg&gt;             Clear all records in the database for this benchmark
-   --create &lt;arg&gt;            Initialize the database for this benchmark
-   --dialects-export &lt;arg&gt;   Export benchmark SQL to a dialects file
-   --execute &lt;arg&gt;           Execute the benchmark workload
--h,--help                          Print this help
-   --histograms                    Print txn histograms
-   --load &lt;arg&gt;              Load data using the benchmark's data loader
--o,--output &lt;arg&gt;            Output file (default System.out)
-   --runscript &lt;arg&gt;         Run an SQL script
--s,--sample &lt;arg&gt;            Sampling window
--v,--verbose                       Display Messages
+```sh
+$ ./tpccbenchmark --create=true --load=true
 ```
 
-Before starting the workload, you will need to load the data first.
+The above command runs the benchmark against the database running at 127.0.0.1 with 10 warehouses.
+To run the benchmark against a specific set of nodes with 100 warehouses:
 
 ```sh
-$ ./tpccbenchmark -c config/workload_all.xml --create=true --load=true
+$ ./tpccbenchmark --create=true --load=true \
+  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 --warehouses=100 --loaderthreads=48
 ```
 
-Then, you can run the workload.
+You can then run the workload.
```

Comment:
Lets make this Step 3, and say "Run TPC-C Benchmark"

Reply:
Done.